src/bots/discord/deploy-commands.ts

```diff
@@ -1,38 +1,26 @@
-import { REST } from '@discordjs/rest';
-import { Routes } from 'discord-api-types/v9';
-import fs from 'fs';
+import { registerCommands } from './utils/registerCommands';
 
 const args = process.argv.slice(2);
-if (args.length != 2) {
-  console.log('deploy-commands accepts two arguments, token and client id ');
+if (args.length != 3) {
+  console.log('deploy-commands accepts three arguments, token, client id and guild id');
```

Comment:
We could skip reading arguments alltogether, right? And instead rely only on `.env`? When I created this script I didn't have access to env variables.

Reply:
Oh. Is this script needed at all even? You register commands on bot startup now, yes? If so, let's fry this file.

Reply:
yes, this can be removed :+1: